src/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -117,10 +117,10 @@ function( $carry, $key ) use ( $directives ) {
 	 */
 	private function get_main_images_html( $context, $product_id ) {
 		$attributes = array(
-			'data-wc-bind--style' => 'selectors.woocommerce.productGalleryLargeImage.styles',
-			'data-wc-effect'      => 'effects.woocommerce.scrollInto',
-			'class'               => 'wc-block-woocommerce-product-gallery-large-image__image',
-			'data-wc-class--wc-block-woocommerce-product-gallery-large-image__image--active-image-slide' => 'selectors.woocommerce.isSelected',
+			'class'                => 'wc-block-woocommerce-product-gallery-large-image__image',
+			'data-wc-bind--hidden' => '!state.isSelected',
+			'data-wc-watch'        => 'callbacks.scrollInto',
+			'data-wc-class--wc-block-woocommerce-product-gallery-large-image__image--active-image-slide' => 'state.isSelected',
```

Comment:
This line was added during https://github.com/woocommerce/woocommerce-blocks/pull/11721/commits/4d6a0b0cd2390c63a2d9bcd79b8e32b8f6968f9c merge, although I'm not sure if it should be kept or not. 🤔

cc: @thealexandrelara

Reply:
Thank you for the ping, yes, it should be there. We are using it to control the currently active image.